lib/src/widgets/rive_animation.dart

```diff
@@ -135,7 +135,9 @@ class _RiveAnimationState extends State<RiveAnimation> {
     // Add any user-created contollers
     widget.controllers.forEach(artboard.addController);
 
+    if(mounted){
     setState(() => _artboard = artboard);
```

Comment:
Can you format the file for consistency?

Reply:
fomated
d1af0da98b0631cfc1435a204a5dff5f900caed9